compiler/rustc_hir/src/hir_id.rs

```diff
@@ -11,13 +12,29 @@ use std::fmt;
 /// the `local_id` part of the `HirId` changing, which is a very useful property in
 /// incremental compilation where we have to persist things through changes to
 /// the code base.
-#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, PartialOrd, Ord)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
 #[derive(Encodable, Decodable)]
 pub struct HirId {
     pub owner: LocalDefId,
     pub local_id: ItemLocalId,
 }
 
+impl Ord for HirId {
```

Comment:
Can you remove the impls for PartialOrd/Ord for HirId too? It compares LocalDefIds, which we want to avoid.

Reply:
On it. 👍